content/docs/tasks/telemetry/gateways/index.md

```diff
@@ -0,0 +1,632 @@
+        kind: VirtualService
+        metadata:
+          name: prometheus-vs
+          namespace: istio-system
+        spec:
+          hosts:
+          - "$TELEMETRY_DOMAIN"
+          gateways:
+          - prometheus-gateway
+          http:
+          - match:
+            - port: 15030
+            route:
+            - destination:
+                host: prometheus
+                port:
+                  number: 9090
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: DestinationRule
+        metadata:
+          name: prometheus
+          namespace: istio-system
+        spec:
+          host: prometheus
+          trafficPolicy:
+            tls:
+              mode: DISABLE
+        ---
+        EOF
+        gateway.networking.istio.io "prometheus-gateway" configured
+        virtualservice.networking.istio.io "prometheus-vs" configured
+        destinationrule.networking.istio.io "prometheus" configured
+        {{< /text >}}
+
+   1. Apply the tracing configuration.
+
+        {{< text bash >}}
+        $ cat <<EOF | kubectl apply -f -
+        apiVersion: networking.istio.io/v1alpha3
+        kind: Gateway
+        metadata:
+          name: tracing-gateway
+          namespace: istio-system
+        spec:
+          selector:
+            istio: ingressgateway
+          servers:
+          - port:
+              number: 15032
+              name: https-tracing
+              protocol: HTTPS
+            tls:
+              mode: SIMPLE
+              serverCertificate: sds
+              privateKey: sds
+              credentialName: telemetry-gw-cert
+            hosts:
+            - "$TELEMETRY_DOMAIN"
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: VirtualService
+        metadata:
+          name: tracing-vs
+          namespace: istio-system
+        spec:
+          hosts:
+          - "$TELEMETRY_DOMAIN"
+          gateways:
+          - tracing-gateway
+          http:
+          - match:
+            - port: 15032
+            route:
+            - destination:
+                host: tracing
+                port:
+                  number: 80
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: DestinationRule
+        metadata:
+          name: tracing
+          namespace: istio-system
+        spec:
+          host: tracing
+          trafficPolicy:
+            tls:
+              mode: DISABLE
+        ---
+        EOF
+        gateway.networking.istio.io "tracing-gateway" configured
+        virtualservice.networking.istio.io "tracing-vs" configured
+        destinationrule.networking.istio.io "tracing" configured
+        {{< /text >}}
+
+1. Visit the telemetry add-ons via your browser.
+
+   * Kiali: `https://$TELEMERY_DOMAIN:15029/`
+   * Prometheus: `https://$TELEMERY_DOMAIN:15030/`
+   * Grafana: `https://$TELEMERY_DOMAIN:15031/`
+   * Tracing: `https://$TELEMERY_DOMAIN:15032/`
+
+### Option 2: Insecure Access (HTTP)
```

Comment:
Please see my comments on Option 1.
Consider using tabs for the options instead of headings: https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#using-tabs

Reply:
I tried tabs in an early version of this page for the content below the option level and ran into issues with embedded text blocks (bad renders). Are embedded yaml text blocks supported in tabs?

Reply:
Yeah, this is the result when I use tabs:

![image](https://user-images.githubusercontent.com/21148125/54163768-4ee3cf80-4417-11e9-8a31-21950e770ae8.png)

Notice all the broken text in what were previously working `{{< text >}}` block.

Reply:
It's somewhat hit or miss with tabs and nested stuff. We're seeing the same problem with the case of a {{<text>}} block in a bullet inside of a {{<warning>}} or {{<tip>}}. 

The grammars just don't nest properly and the parsers are stepping on one another. The key challenge is that markdown gives meaning to indents and spacing. As data is transformed in the Hugo pipeline, chunks of data suddenly gets treated as a code block (due to indent) leading to bad things downstream.